Sources/StreamVideoSwiftUI/ViewFactory.swift

```diff
@@ -149,27 +149,44 @@ public protocol ViewFactory: AnyObject {
         callSettings: Binding<CallSettings>,
         call: Call?
     ) -> LocalParticipantViewModifierType
+
+    associatedtype BottomParticipantsBarLayoutComponentType: View
+    func makeBottomParticipantsBarLayoutComponent(
+        participants: [CallParticipant],
+        availableFrame: CGRect,
+        call: Call?,
+        onChangeTrackVisibility: @escaping (CallParticipant, Bool) -> Void
+    ) -> BottomParticipantsBarLayoutComponentType
+
+    associatedtype DominantSpeakerLayoutComponentType: View
+    func makeDominantSpeakerLayoutComponent(
```

Comment:
Similarly here, this is just a big view that presents a video. If you want to customize it, you can use the video component customization.